<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exit_push">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iocdetails_push">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>IOC...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FS Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>v.0.1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Phase Cavities</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SXR 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SXR 2</string>
    </property>
   </widget>
   <widget class="PyDMLabel" name="sxrpcav1val">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="precision" stdset="0">
     <number>0</number>
    </property>
    <property name="showUnits" stdset="0">
     <bool>true</bool>
    </property>
    <property name="channel" stdset="0">
     <string>ca://fsioc:pcav1PV</string>
    </property>
    <property name="displayFormat" stdset="0">
     <enum>PyDMLabel::Decimal</enum>
    </property>
   </widget>
   <widget class="PyDMLabel" name="sxrpcav2val">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="showUnits" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>HXR 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>HXR 2</string>
    </property>
   </widget>
   <widget class="PyDMLabel" name="hxrpcav1val">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="showUnits" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="PyDMLabel" name="hxrpcav2val">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="showUnits" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>These are the processed outputs of the phase cavity system. These serve as the input for the cable stabilizer drift correction feedback.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Processed PCAV Data (Plotting not implemented yet)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cable Stabilizer Phase Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FEH Offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NEH Offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>NEH Gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>FEH Gain</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="fehfboffset_val">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="nehfboffset_val">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="fehfbgain_val">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="nehfbgain_val">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="PyDMPushButton" name="zerooffsets_push">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string>Zero Offsets</string>
   </property>
   <property name="text">
    <string>Zero Offsets</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Write current pcav values as new offset</string>
   </property>
  </widget>
  <widget class="PyDMEnumComboBox" name="fehfbenable_ctl">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="maxCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="PyDMEnumComboBox" name="nehfbenable_ctl">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="maxCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webEngineView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webEngineView_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="PyDMRelatedDisplayButton" name="help_push">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Help...</string>
   </property>
   <property name="filenames" stdset="0">
    <stringlist>
     <string>fsmanager_gui_help.ui</string>
    </stringlist>
   </property>
   <property name="openInNewWindow" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable/Disable</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>pydm.widgets.enum_combo_box</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMRelatedDisplayButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.related_display_button</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
